clar2wasm/src/bin/utils/mod.rs

```diff
@@ -0,0 +1,82 @@
+use clap::builder::PossibleValue;
+use clap::ValueEnum;
+use clarity::types::StacksEpochId;
+use clarity::vm::ClarityVersion;
+
+#[derive(Clone)]
+pub struct WrappedEpochId(StacksEpochId);
+
+impl Default for WrappedEpochId {
+    fn default() -> WrappedEpochId {
+        WrappedEpochId(StacksEpochId::latest())
+    }
+}
+
+impl From<WrappedEpochId> for StacksEpochId {
+    fn from(epoch: WrappedEpochId) -> Self {
+        epoch.0
+    }
+}
+
+impl ValueEnum for WrappedEpochId {
+    fn value_variants<'a>() -> &'a [Self] {
+        &[
+            WrappedEpochId(StacksEpochId::Epoch10),
+            WrappedEpochId(StacksEpochId::Epoch20),
+            WrappedEpochId(StacksEpochId::Epoch2_05),
+            WrappedEpochId(StacksEpochId::Epoch21),
+            WrappedEpochId(StacksEpochId::Epoch22),
+            WrappedEpochId(StacksEpochId::Epoch23),
+            WrappedEpochId(StacksEpochId::Epoch24),
+            WrappedEpochId(StacksEpochId::Epoch25),
+            WrappedEpochId(StacksEpochId::Epoch30),
+        ]
+    }
+
+    fn to_possible_value(&self) -> Option<PossibleValue> {
+        match &self.0 {
+            StacksEpochId::Epoch10 => Some(PossibleValue::new("1.0")),
+            StacksEpochId::Epoch20 => Some(PossibleValue::new("2.0")),
+            StacksEpochId::Epoch2_05 => Some(PossibleValue::new("2.05")),
+            StacksEpochId::Epoch21 => Some(PossibleValue::new("2.1")),
+            StacksEpochId::Epoch22 => Some(PossibleValue::new("2.2")),
+            StacksEpochId::Epoch23 => Some(PossibleValue::new("2.3")),
+            StacksEpochId::Epoch24 => Some(PossibleValue::new("2.4")),
+            StacksEpochId::Epoch25 => Some(PossibleValue::new("2.5")),
+            StacksEpochId::Epoch30 => Some(PossibleValue::new("3.0")),
+        }
+    }
+}
+
+#[derive(Clone)]
+pub struct WrappedClarityVersion(ClarityVersion);
+
+impl Default for WrappedClarityVersion {
+    fn default() -> WrappedClarityVersion {
+        WrappedClarityVersion(ClarityVersion::latest())
```

Comment:
Same remark as for the epoch.

Reply:
Went ahead and used `Epoch25` as a default